builder/bundles.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+var pkgMutex sync.RWMutex
```

Comment:
Can you move this declaration closer to the function that uses it?

Reply:
yes